Dockerfile

```
@@ -4,7 +4,7 @@
 #
 # Author: muxator
 
-FROM node:14-buster-slim
+FROM node:slim
```

I would prefer to track the active LTS release because the [Node.js releases page](https://nodejs.org/en/about/releases/) says, "Production applications should only use Active LTS or Maintenance LTS releases."

```suggestion
FROM node:lts-slim
```


Also, Node.js v15 and later come with npm v7, which is a major change over npm v6. We've had some compatibility problems with npm v7 when installing plugins. I recommend changing line 88 to pass `--legacy-peer-deps` to `npm install`:

```diff
diff --git a/Dockerfile b/Dockerfile
--- a/Dockerfile
+++ b/Dockerfile
@@ -85,7 +85,7 @@ COPY --chown=etherpad:etherpad ./ ./
 # seems to confuse tools such as `npm outdated`, `npm update`, and some ESLint
 # rules.
 RUN { [ -z "${ETHERPAD_PLUGINS}" ] || \
-      npm install --no-save ${ETHERPAD_PLUGINS}; } && \
+      npm install --no-save --legacy-peer-deps ${ETHERPAD_PLUGINS}; } && \
     src/bin/installDeps.sh && \
     rm -rf ~/.npm
 
```

---

I agree

---

Do you think we should add an apt update && apt upgrade -y in the Dockerfile as well?

---

Updating the base image packages should be the responsibility of the Debian image maintainers, and the Node.js image maintainers should be regularly refreshing to get the latest Debian image. With that said, we're already running `apt-get update` so I'm OK with inserting an `apt-get -qq dist-upgrade` after that.

Don't forget to pass `--legacy-peer-deps` to `npm install` on line 88.